modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/actionItems/filterCreationActions.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ICreationActionItem} from '../../management_bar/components/CreationMenu';
+
+const filterCreationActions = ({
+	customActions,
```

Comment:
Rename to `customActions` as it could cover bulkActions. Maybe is too early to think about this

Reply:
I am not sure if this will cover bulk actions, because bulk actions are based on item actions, and this has code pretty specific to collection actions. Bulk actions are basically a way of doing the same item action, but on multiple items.

Reply:
Another point for my reasoning is TS in this PR:
```ts
const filterCreationActions = ({creationActions, globalCollectionActions}: {
	customActions: Array<ICreationActionItem>;
	globalCollectionActions: any
}): Array<ICreationActionItem> | null => {
```
It's not 
```
customActions: Array<ICreationActionItem> | undefined, // or null
```

Reply:
Global actions include a bunch of `*Batch` permission keys. It is just a guess

![image](https://github.com/liferay-frontend/liferay-portal/assets/132653342/3efaed8d-5f6d-47ba-bca7-a0e263335e59)